internal/staticanalysis/parsing/js/babel-parser.js

```diff
@@ -182,19 +198,32 @@ function findLiteralsAndIdentifiers(source, printDebug) {
 }
 
 function main() {
-    const printDebug = false;
+    const syntaxErrorExitCode = 33;
```

Comment:
nit: use SYNTAX_ERROR_EXIT_CODE per https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
Will fix in another PR